### PR TITLE
Added a fix for the checksum of sha256sum

### PR DESCRIPTION
**Pull request by pmburu**

### CODE REVIEW

**Review by sundowndev — CHANGES_REQUESTED** [review]
Thank you for this fix 🎉 

Just left a small comment to override any existing occurrence of `PhoneInfoga`.

File: support/scripts/install [hunk 1]
```diff
@@ -11,14 +11,14 @@ if [ $os == "Linux_x86_64" ] || [ $os == "Linux_armv6" ] || [ $os == "Darwin_x86
   echo "Found version $phoneinfoga_version"
 
   echo "Downloading version $phoneinfoga_version..."
-  wget "https://github.com/sundowndev/phoneinfoga/releases/download/$phoneinfoga_version/PhoneInfoga_$os.tar.gz"
+  wget "https://github.com/sundowndev/phoneinfoga/releases/download/$phoneinfoga_version/phoneinfoga_$os.tar.gz"
 
   echo "Verifying checksum..."
-  curl -sSL https://github.com/sundowndev/phoneinfoga/releases/download/$phoneinfoga_version/PhoneInfoga_checksums.txt | sha256sum -c --strict --ignore-missing
+  echo "$(curl -sSL https://github.com/sundowndev/phoneinfoga/releases/download/$phoneinfoga_version/PhoneInfoga_checksums.txt) phoneinfoga_$os.tar.gz" | sha256sum --strict --check --ignore-missing
```
[the remark this answers is not in the document]
Comment (pmburu) [reply support/scripts/install:17]: If we touch the txt file name on the script, we have to rename it on the release as well. Would you like me to rename it?

[the remark this answers is not in the document]
Comment (sundowndev) [reply support/scripts/install:17]: It's already in lowercase in the release, isn't it ? https://github.com/sundowndev/phoneinfoga/releases

[the remark this answers is not in the document]
Comment (pmburu) [reply support/scripts/install:17]: Hi @sundowndev, did you need me to do anything else to the project? Thanks

[the remark this answers is not in the document]
Comment (sundowndev) [reply support/scripts/install:17]: Yes, please use `phoneinfoga_checksums` here

**Review by pmburu — COMMENTED** [review]
Hello @sundowndev , I made the changes. Please have a look at it and let me know whether it is okay.

File: support/scripts/install (hunk 1 above)
[the remark this answers is not in the document]
Comment (pmburu) [reply support/scripts/install]: @sundowndev, it is done now. Please have a look at it and let me know whether it is aligned thanks.

**Review by sundowndev — APPROVED** [review]
Thank you 🙏🏻 ❤️

**Pull request merged**